src/MaterialEditor.Core.Studio/Core.MaterialEditor.Studio.cs

```diff
@@ -15,6 +15,7 @@
 using UnityEngine;
 using UnityEngine.SceneManagement;
 using UnityEngine.UI;
+using static Illusion.Utils;

```

Comment:
This is dangerous, different games have different stuff in here that might hide unexpected things.

Reply:
accidentally added when trying stuff I think. Actually unused and will remove it again